lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1246,12 +1246,10 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
     case ResourceNodeType::DescriptorTableVaPtr: {
       // Check if the descriptor set is in use. For compute with calls, enable it anyway.
       UserDataNodeUsage *descSetUsage = nullptr;
-      if (!isComputeWithCalls()) {
         if (userDataUsage->descriptorTables.size() > userDataNodeIdx)
```

Comment:
hi @Flakebi ! could you remove indent of the four lines starting from this line to make them align with other  lines?

Reply:
Done. I’m surprised that the formatter didn’t pick that up.